I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 27, in script call
    m "zzz..."
  File "renpy/common/00console.rpy", line 765, in script call
    python in _console:
  File "renpy/common/00start.rpy", line 120, in script
    return
Exception: Could not find return label (u'C:\\renpy\\The Good Christian Visual Novel/game/script.rpy', 1540666498, 14).

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 27, in script call
    m "zzz..."
  File "renpy/common/00console.rpy", line 765, in script call
    python in _console:
  File "renpy/common/00start.rpy", line 120, in script
    return
  File "C:\renpy\renpy-6.99.14.3-sdk\renpy\ast.py", line 1468, in execute
    next_node(renpy.game.context().lookup_return(pop=True))
  File "C:\renpy\renpy-6.99.14.3-sdk\renpy\execution.py", line 664, in lookup_return
    raise Exception("Could not find return label {!r}.".format(self.return_stack[-1]))
Exception: Could not find return label (u'C:\\renpy\\The Good Christian Visual Novel/game/script.rpy', 1540666498, 14).

Windows-8-6.2.9200
Ren'Py 7.1.1.929
The Good Christian Visual Novel 1.0
Sat Oct 27 13:57:21 2018
